src/Ookii.Dialogs.WinForms/Ookii.Dialogs.WinForms.csproj

```diff
@@ -1,39 +1,28 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProductVersion>9.0.30729</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{779E62B3-C5A6-494D-9D87-96C0DEC7FFB0}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Ookii.Dialogs.WinForms</RootNamespace>
-    <AssemblyName>Ookii.Dialogs.WinForms</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <TargetFrameworkProfile />
+    <TargetFrameworks>net45;netcoreapp3.1;net5.0-windows</TargetFrameworks>
+    <AssemblyTitle>Ookii.Dialogs.WinForms</AssemblyTitle>
+    <Company>augustoproiete.net</Company>
+    <Product>Ookii.Dialogs.WinForms</Product>
+    <Description>Ookii.Dialogs.WinForms is a class library for .NET applications providing several common dialogs. Included are classes for task dialogs, credential dialog, progress dialog, input dialog, folder browser dialog, and more.</Description>
+    <Copyright>Copyright (c) 2009-2021 Ookii Dialogs Contributors</Copyright>
+    <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
+    <Version>0.0.1-local</Version>
+    <DocumentationFile>bin\$(Configuration)\Ookii.Dialogs.WinForms.xml</DocumentationFile>
+    <OutputPath>bin\$(Configuration)\</OutputPath>
+    <Authors>Ookii Dialogs Contributors</Authors>
+    <PackageIconUrl>https://raw.githubusercontent.com/augustoproiete/ookii-dialogs-winforms/master/assets/ookii-dialogs-winforms-nuget.png</PackageIconUrl>
+    <PackageId>Ookii.Dialogs.WinForms</PackageId>
+    <PackageProjectUrl>https://github.com/augustoproiete/ookii-dialogs-winforms</PackageProjectUrl>
```

Comment:
The URL of the project changed recently to `https://github.com/ookii-dialogs/ookii-dialogs-winforms`

Reply:
This should now have been changed to indicate the correct URL.